src/services/notion.service.ts

```diff
@@ -297,6 +297,36 @@ export async function createFuelRecord(data: Fuel) {
   }
 }
 
+export async function createBillRecord(data: Bill) {
+  try {
+    // Append the FuelFormID as a query parameter to the URL
+    const response = await fetch(
+      `/api/notion/database?databaseId=${NOTION_DATABASES.BILLS}`,
+      {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(parseObjectToNotion<Bill>(data)),
+      }
+    );
+    const json = await response.json();
+    if (response.status !== 200) {
+      toast.error("Something went wrong" + response.statusText);
+      return false;
+    }
+    if (json.error) {
+      toast.error("Something went wrong" + json.error);
+      return false;
+    }
+
+    return parseNotionObject<Bill>(new Bill(), json.result as NotionPage);
+  } catch (error) {
+    console.error("Error retrieving page from Notion:", error);
+    return undefined;
```

Comment:
In case of error is better to return an object {errror: "error"} 
insteaad of just false
So in the client we can display thereal error

Otherwise in case of error the client has no idea what was wrong (hence you have no idea what to fix when the bug comes)

The client can display this error in a modal or other way